model/task_lifecycle.go

```diff
@@ -90,6 +90,20 @@ func SetActiveState(t *task.Task, caller string, active bool) error {
 	} else {
 		return nil
 	}
+	if !active && t.Requester == evergreen.MergeTestRequester {
```

Comment:
I would think this should also get hit by the case at 79.

Reply:
The intent of this is to catch cases where the user is the one unscheduling, so I don't believe it'll hit the case above

Reply:
Actually after re-reading maybe this is more what I meant:
Of the if, else if, and else cases, only the if else case would even check this (because the if case is only for active, and the else case returns early). So really this should be part of the else if at 79. The only reason there is this piece of code after the if/elses is because for both modified cases, we need to update the cache.

Reply:
got it, fixed